src/Browser.ts

```diff
@@ -22,7 +22,7 @@ import Protocol from './protocol';
 import { Connection } from './Connection';
 import { Page } from './Page';
 import { ChildProcess } from 'child_process';
-import type { Viewport } from './PuppeteerViewport';
+import { Viewport } from './PuppeteerViewport';
```

Comment:
API Extractor doesn't work fully with `import type` yet so we should use regular imports. This isn't really an issue for us anyway.